src/services/review/RepoCheckerService.ts

```diff
@@ -177,6 +493,60 @@ export default class RepoCheckerService {
     )
   }
 
+  outputHumanReadableErrors(repoName: string) {
+    const csvData = fs.readFileSync(
+      path.join(SITE_CHECKER_REPO_PATH, repoName, "log.csv"),
+      "utf8"
+    )
+
+    const report: string[] = []
+
+    Papa.parse(csvData, {
+      header: true,
+      complete(results) {
+        results.data.forEach((row: any) => {
```

Comment:
please add a type here - we can just cast but having `any` is prone to error and doesn't preserve information